.github/workflows/markdown-link-check.yml

```diff
@@ -5,14 +5,17 @@ on:
 
 permissions:
   contents: read
+  pull-requests: write
 
 jobs:
   check-links:
     name: Check links
     runs-on: ubuntu-latest
     timeout-minutes: 10
     steps:
-    - uses: actions/checkout@main
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
-      with:
-        base-branch: 'main'
+      - uses: actions/checkout@v4
+      - name: Run linkspector
+        uses: umbrelladocs/action-linkspector@v1
+        with:
+          reporter: github-pr-review
+          fail_on_error: true
```

Comment:
It is convenient that files end with an empty line. So just add one.

Reply:
Thanks, added an empty line. Just found out why: https://stackoverflow.com/a/729795